### PR TITLE
test: Add tests for TagAddNewTagModal component

### PR DESCRIPTION
**Pull request by Merkur39**
Ajout des tests suite aux retours https://github.com/cozy/cozy-banks/pull/2467

### CODE REVIEW

**Comment by Merkur39** [comment]
[[BUG] Il est possible de créer un label vide](https://trello.com/c/5Q2FSrPp/1939-bug-il-est-possible-de-cr%C3%A9er-un-label-vide)

File: src/components/Tag/TagAddNewTagModal.jsx [hunk 1]
```diff
@@ -63,6 +64,7 @@ const TagAddNewTagModal = ({ onClick, onClose, withLabel }) => {
             busy={isBusy}
             disabled={label.length === 0}
             onClick={handleClick}
+            data-testid="TagAddNewTagModal-Button-submit"
```
Comment (trollepierre) [inline src/components/Tag/TagAddNewTagModal.jsx:67]: is it mandatory to use data-testid only for test purpose?

isn't it possible to use getByRole('button') or getAllByRole('button')[0] ?

File: src/components/Tag/TagAddNewTagModal.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    const newTagInput = getByTestId('TagAddNewTagModal-TextField')
+    const submitButton = getByTestId('TagAddNewTagModal-Button-submit')
+
+    fireEvent.change(newTagInput, { target: { value: '' } })
+    fireEvent.click(submitButton)
+
+    expect(mockSave).toBeCalledTimes(0)
+  })
+
+  it('should remove all spaces before & after the words before saving them', () => {
```
Comment (trollepierre) [inline src/components/Tag/TagAddNewTagModal.spec.jsx:52]:
```suggestion
  it('should trim spaces before saving them', () => {
```

**Review by trollepierre — APPROVED** [review]

File: src/components/Tag/TagAddNewTagModal.jsx (hunk 1 above)
Comment (Merkur39) [reply src/components/Tag/TagAddNewTagModal.jsx:67]: > nit: is it mandatory to use data-testid only for test purpose?

Pas obligatoire, mais c'est beaucoup plus clair ainsi 👍

**Pull request merged**